cogs/index_service_cog.py

```diff
@@ -12,6 +12,7 @@
 USER_INPUT_API_KEYS = EnvService.get_user_input_api_keys()
 USER_KEY_DB = EnvService.get_api_db()
 PRE_MODERATE = EnvService.get_premoderate()
+GITHUB_TOKEN = EnvService.get_github_token()
```

Comment:
We want to propagate this to the environment after it's retrieved, with `os.setenv` or something.
So how the github reader token retrieval works is it looks in the program/server/computer runtime environment, and since the github token is loaded only into the `.env`, we need to manually load it into the program/server/computer runtime environment